pkg/kubectl/rolling_updater.go

```diff
@@ -763,6 +759,33 @@ func updateWithRetries(rcClient client.ReplicationControllerInterface, rc *api.R
 	return rc, err
 }
 
+type updateFuncForPod func(pod *api.Pod)
+
+// updateWithRetriesForPod updates applies the given pod as an update.
+func updateWithRetriesForPod(podClient client.PodInterface, pod *api.Pod, applyUpdate updateFuncForPod) (*api.Pod, error) {
+	var err error
+	oldPod := pod
+	err = wait.Poll(10*time.Millisecond, 1*time.Minute, func() (bool, error) {
```

Comment:
10ms is a short interval but it's fine if we backoff. Generally retries should backoff.